specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefitutilization.json

```diff
@@ -0,0 +1,504 @@
+      "get": {
+        "tags": [
+          "BenefitUtilizationSummaries"
+        ],
+        "operationId": "BenefitUtilizationSummaries_ListBySavingsPlanId",
+        "description": "Lists the savings plan utilization summaries for daily or monthly grain.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/cost-management/"
+        },
+        "x-ms-examples": {
+          "SavingsPlanUtilizationSummariesMonthlyWithSavingsPlanId": {
+            "$ref": "./examples/BenefitUtilizationSummaries/SavingsPlan-SavingsPlanId-Monthly.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "common-types.json#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "$filter",
+            "description": "Supports filtering by usageDate.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/grainParameter"
+          },
+          {
+            "$ref": "#/parameters/savingsPlanIdParameter"
+          },
+          {
+            "$ref": "#/parameters/savingsPlanOrderIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BenefitUtilizationSummariesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BenefitUtilizationSummariesListResult": {
+      "description": "Result of listing benefit utilization summaries.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of benefit utilization summaries.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/BenefitUtilizationSummary"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BenefitUtilizationSummary": {
+      "description": "Benefit utilization summary resource.",
+      "discriminator": "kind",
+      "type": "object",
+      "required": [
+        "kind"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "description": "Reservation, SavingsPlan or IncludedQuantity.",
```

Comment:
>Reservation [](http://example.com/codeflow?start=26&length=11)

Can quote them to be clearer that these are a list o the allowed values:
'Reservation', 'SavingsPlan' or 'IncludedQuantity'